help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,219 @@
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your prospects, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Outreach], to display in the prospects’ feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform Prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Outreach] destination, you must have a [schema](/help/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
+
+### Outreach Prerequisites {#prerequisites-destination}
+
+Note the following prerequisites in [!DNL Outreach], in order to export data from Platform to your [!DNL Outreach] account:
+
+#### You need to have an Outreach account {#prerequisites-account}
+
+Go to the [!DNL Outreach] [sign in](https://accounts.outreach.io/users/sign_in) page to register and create an account, if you do not have one already. Also see the [!DNL Outreach] support [page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account) for more details.
+
+#### Gather Outreach credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Outreach] CRM destination:
+
+| Credential | Description |
+|---|---|
+| Email | Your [!DNL Outreach] account email |
+| Password | Your [!DNL Outreach] account password |
+
+#### Setup custom field labels
+
+[!DNL Outreach] supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the labels to their corresponding segment names instead of keeping the defaults. For example as below:
+
+[!DNL Outreach] settings page for prospects displaying custom fields.
+![[!DNL Outreach] custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+[!DNL Outreach] settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![[!DNL Outreach] custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
```

Comment:
One thing that applies to all destinations. To improve readability and accessibility for all our docs, we have been asked to use clearer alt-text for images. Please review all the alt-text in the page and update it to a form similar to this:

"Outreach UI screenshot showing custom fields with associated labels."

Use clear sentences that describe as accurate as possible what is shown in the image. This is especially useful for users who are unable to see the image and/or use screen readers.